components/ADempiere/FileRender/ExcelFile/index.vue

```diff
@@ -106,10 +106,15 @@ export default defineComponent({
     }
 
     function generateReaderData() {
+      console.log({
+        mimeType: props.mimeType,
+        outputStream: props.stream
+      })
       const { blobFile, dataValues } = buildBlobAndValues({
         mimeType: props.mimeType,
         outputStream: props.stream
       })
+      console.log({ blobFile, dataValues })
```

Comment:
@elsiosanchez please remove this console

Reply:
Ready. console removed